modules/tracking/test/test_trackers.cpp

```diff
@@ -567,6 +567,22 @@ TEST_P(DistanceAndOverlap, Scaled_Data_CSRT)
   test.run();
 }
 
+TEST(GOTURN, memory_usage)
+{
+  cv::Rect2d roi(145, 70, 85, 85);
+  cv::Mat frame;
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
+  string folder = cvtest::TS::ptr()->get_data_path() + "tracking/david/data/david.webm";
```

Comment:
Please add a tracker to https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/download_models.py

Reply:
Which source should I add?

Reply:
the links to download GOTURN's prototxt and caffemodel - after that we can add it to buildbot

Reply:
I understand that I should provide links to GOTURN.
I asking about which one - is it OK, if it is gonna be links to my google drive, or it should be another source?

Reply:
I am sorry for inconvenience, but can you please tell me, what is ```sha``` and where I can found it for these links/files?
Example:
```python
    Model(
        name='YOLOv4-tiny',  # https://github.com/opencv/opencv/issues/17148
        url='https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-tiny.weights',
        sha='d110379b7b86899226b591ad4affc7115f707157',
        filename='yolov4-tiny.weights')
```

Reply:
Added here: https://github.com/opencv/opencv_extra/pull/795 (you can make a copy of PR branch into your opencv_extra for testing)

Reply:
Model is successfully downloaded using ```download_models.py```.